server/service/frontend.go

```diff
@@ -68,6 +70,71 @@ func ServeFrontend(urlPrefix string, sandbox bool, logger log.Logger) http.Handl
 	})
 }
 
+func ServeEndUserEnroll(urlPrefix string, logger log.Logger) http.Handler {
+	herr := func(w http.ResponseWriter, err string) {
+		logger.Log("err", err)
+		http.Error(w, err, http.StatusInternalServerError)
+	}
+
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		writeBrowserSecurityHeaders(w)
+
+		fs := newBinaryFileSystem("/frontend")
+		file, err := fs.Open("templates/byod.tmpl")
+		if err != nil {
+			fmt.Print("test")
+			logger.Log("err", err)
+			http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
we ended up using `herr` for all the others after all, let's use it here too

Reply:
added